### PR TITLE
Fix for the transformers translation plus some renaming

### LINKED ISSUE #160: Address PM comments 2024-01-24
Refer to the [meeting notes](https://exasol.atlassian.net/wiki/spaces/InTeam/pages/504332304/2024-01-24+AI+Lab+Sync)

### LINKED ISSUE #161: Need to specify languages when calling the Transformers Translation model

### PR DESCRIPTION
**Pull request by ahsimb**
closes #160
closes #161

### CODE REVIEW

File: doc/changes/changes_0.1.0.md
Comment (ckunki) [inline doc/changes/changes_0.1.0.md:1]: Yesterday TK and me finally shipped somewhat like an initial release :rocket: :tada:.
Please 
* decide for a new version number, e.g.  0.2.0 or 1.0.0
*  increase the version number in file `pyproject.toml`.
* create a new changes file, e.g. `changes_0.2.0.md` or `changes_1.0.0.md` and move your additions there. 
*  update file `changelog.md`, too.

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/text_generation.ipynb [hunk 1]
```diff
@@ -130,7 +130,7 @@
    "id": "27b1dd67-ffed-4bf8-9ee7-a1e003cdbcc6",
    "metadata": {},
    "source": [
-    "We will be updating this variable at every call to the model.\n",
+    "At the start, the `MY_TEXT` variable has an initial context. We will be updating and printing this variable at every call to the model.\n",
```
Comment (ckunki) [inline exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/text_generation.ipynb:133]:
```suggestion
    "At the start, the `MY_TEXT` variable has an initial context. We will update and print this variable at every call to the model.\n",
```

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/sequence_classification.ipynb [hunk 2]
```diff
@@ -100,7 +100,8 @@
    "source": [
     "## Use language model\n",
     "\n",
-    "Let's try to classify a single phrase which definitely bears emotions but is also somewhat ambiguous - \"Oh my God!\""
+    "Let's try to classify a single phrase that definitely bears emotions but is also somewhat ambiguous - \"Oh my God!\".\n",
+    "We will save the result in the variable `udf_output` to perform automatic testing of this notebook."
```
Comment (ckunki) [inline exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/sequence_classification.ipynb:104]:
```suggestion
    "We will save the result in the variable `udf_output` to support automatic testing of this notebook."
```

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/question_answering.ipynb [hunk 3]
```diff
@@ -130,7 +130,8 @@
    "id": "43f3f6e9-6f40-49b2-bbba-8954c35b5e06",
    "metadata": {},
    "source": [
-    "We will collect 5 best answers."
+    "We will collect 5 best answers.\n",
+    "We will save the result in the variable `udf_output` to perform automatic testing of this notebook."
```
Comment (ckunki) [inline exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/question_answering.ipynb:134]:
```suggestion
    "We will save the result in the variable `udf_output` to support automatic testing of this notebook."
```

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/masked_modelling.ipynb [hunk 4]
```diff
@@ -121,7 +121,8 @@
    "id": "f3727075-4680-4f2f-83c2-17e5c22c6a57",
    "metadata": {},
    "source": [
-    "We will collect 5 best answers."
+    "We will collect the 5 best answers.\n",
+    "We will save the result in the variable `udf_output` to perform automatic testing of this notebook."
```
Comment (ckunki) [inline exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/masked_modelling.ipynb:125]:
```suggestion
    "We will save the result in the variable `udf_output` to support automatic testing of this notebook."
```

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/token_classification.ipynb [hunk 5]
```diff
@@ -131,6 +131,14 @@
     "MY_TEXT = MY_TEXT.replace(\"'\", \"''\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "dff2b9c8-6b6a-4e7a-b065-898d841a34fc",
+   "metadata": {},
+   "source": [
+    "Let's run the token classification model. We will save the result in the variable `udf_output` to perform automatic testing of this notebook."
```
Comment (ckunki) [inline exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/token_classification.ipynb:139]:
```suggestion
    "Let's run the token classification model. We will save the result in the variable `udf_output` to support automatic testing of this notebook."
```

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/translation.ipynb [hunk 6]
```diff
@@ -115,7 +115,20 @@
...
+    "\n",
+    "MAX_TRANSLATION_LENGTH = 200"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cb8641d1-1b84-47a5-8ef0-e1f01cd8a412",
+   "metadata": {},
+   "source": [
+    "Let's run the translation model. We will save the result in the variable `udf_output` to perform automatic testing of this notebook."
```
Comment (ckunki) [inline exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/translation.ipynb:131]:
```suggestion
    "Let's run the translation model. We will save the result in the variable `udf_output` to support automatic testing of this notebook."
```

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/zero_shot_classification.ipynb [hunk 7]
```diff
@@ -131,6 +131,14 @@
     "MY_LABELS = 'space & cosmos, scientific discovery, microbiology, robots, archeology'"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "768a97ed-a104-4f89-8c7b-9fdd6558b5ab",
+   "metadata": {},
+   "source": [
+    "Let's run the zero shot text classification model. We will save the result in the variable `udf_output` to perform automatic testing of this notebook."
```
Comment (ckunki) [inline exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/zero_shot_classification.ipynb:139]:
```suggestion
    "Let's run the zero shot text classification model. We will save the result in the variable `udf_output` to support automatic testing of this notebook."
```

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/main_config_ui.ipynb [hunk 8]
```diff
@@ -44,28 +44,28 @@
...
-    "    missing = 'Docker-DB is NOT CREATED'\n",
+    "    running = 'Exasol Docker-DB is RUNNING'\n",
+    "    stopped = 'Exasol Docker-DB is STOPPED'\n",
+    "    missing = 'Exasol Docker-DB is NOT CREATED'\n",
     "\n",
     "\n",
     "def get_db_selection_ui() -> widgets.Widget:\n",
     "    \"\"\"\n",
-    "    Creates a UI form for choosing between the Docker-DB and a customer database.\n",
+    "    Creates a UI form for choosing between the Exasol Docker-DB and an external Exasol Database.\n",
```
Comment (ckunki) [inline exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/main_config_ui.ipynb:56]: :+1:

**Review by ckunki — CHANGES_REQUESTED** [review]

File: doc/changes/changes_0.1.0.md
Comment (tkilias) [reply doc/changes/changes_0.1.0.md:1]: 0.1.1 or 0.2.0 are next, the 1.0.0 will be end of the month

**Comment by ahsimb** [comment]
JupySQL's official documentation is inadequate. 
I think it's good to have the --save option for two reasons. Firstly, it could be helpful for the user. It works like the `WITH` clause. You can use the saved result as a table. Secondly, inserting it during a test will be too hacky for my liking. It will limit the format of the query to a certain pattern. On the other hand, if we don't mention the test the user might decide that she doesn't need it and delete it for better clarity.

**Review by tkilias — APPROVED** [review]

**Review by ckunki — APPROVED** [review]

**Pull request merged**